src/bitops.c

```diff
@@ -504,7 +504,9 @@ robj *lookupStringForBitCommand(client *c, size_t maxbit) {
  * If the source object is NULL the function is guaranteed to return NULL
  * and set 'len' to 0. */
 unsigned char *getObjectReadOnlyString(robj *o, long *len, char *llbuf) {
-    serverAssert(o->type == OBJ_STRING);
+    if (o) {
+        serverAssert(o->type == OBJ_STRING);
+    }
```

Comment:
I think this is cleaner:
```suggestion
    serverAssert(!o || o->type == OBJ_STRING);
```